app/views/models/new.html.erb

```diff
@@ -32,6 +32,11 @@
         <%= text_area_tag 'model[description]', model_description_value, :class=>"form-control rich-text-edit" -%>
       </div>
 
+       <%= fields_for(@model) do |model_fields| %>
+        <%= render :partial => 'extended_metadata/extended_metadata_type_selection', :locals => {:f=> model_fields,:resource => @model } %>
+        <%= render :partial => 'extended_metadata/extended_metadata_attribute_input', :locals => {:f=> model_fields, :resource => @model } %>
+      <% end %>
+
```

Comment:
use the format `partial:` rather than `:partial=>` , and the same for other keys